src/CommunityToolkit/Xamarin.CommunityToolkit/Views/SegmentedView/iOS/SegmentedViewRenderer.ios.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Linq;
+using Xamarin.CommunityToolkit.iOS.UI.Views;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using UIKit;
+using Xamarin.Forms.Platform.iOS;
+using System.Threading.Tasks;
+using System.Threading;
+using Foundation;
+using Xamarin.Forms.Internals;
+using Xamarin.CommunityToolkit.Extensions.iOS;
+
+[assembly: ExportRenderer(typeof(SegmentedView), typeof(SegmentedViewRenderer))]
+
+namespace Xamarin.CommunityToolkit.iOS.UI.Views
+{
+	public class SegmentedViewRenderer : ViewRenderer<SegmentedView, UISegmentedControl>
+	{
+		const string TAG = "SegmentedView";
+
+		protected override async void OnElementChanged(ElementChangedEventArgs<SegmentedView> e)
+		{
+			base.OnElementChanged(e);
+
+			if (e.NewElement == null)
+				return;
+
+			if (Control == null)
+				SetNativeControl(new UISegmentedControl());
+
+			if (e.OldElement != null && Control != null)
+				InvalidateControl();
+
+			if (e.NewElement != null)
+			{
+				try
+				{
+					await InitializeControl();
+				}
+				catch (Exception ex)
+				{
+					Log.Warning(TAG, ex.Message);
+				}
+			}
+		}
+
+		async Task InitializeControl()
+		{
+			await PopulateSegments(Element.Items);
+			Control.ClipsToBounds = true;
+			Control.SelectedSegment = Element.SelectedIndex;
+			Control.Layer.MasksToBounds = true;
+			Control.BackgroundColor = Element.BackgroundColor.ToUIColor();
+			UpdateSelectedSegment(Element.SelectedIndex);
+
+			Control.ValueChanged += OnSelectedIndexChanged;
```

Comment:
Same for control (?)
```suggestion
                        var control = Control;
			control.ValueChanged += OnSelectedIndexChanged;
```

Reply:
hmm... What will this get me? Are you suggesting using local variable for the event sub only or in general?